model/host/host.go

```diff
@@ -811,3 +813,38 @@ func (h *Host) GetParent() (*Host, error) {
 
 	return host, nil
 }
+
+// AggregateLastContainerFinishTimes returns the latest finish time for each host with containers
```

Comment:
I think we've let this comment style slip through and should have caught it earlier, sorry. Don't use newlines to indicate logical breaks. I think you can delete "errors if aggregation fails" altogether, since this is the convention. You can do the same with the next function.